include/drivers/gpio.h

```diff
@@ -702,11 +757,15 @@ static inline int z_impl_gpio_port_get_raw(struct device *port,
  * Value of a pin with index n will be represented by bit n in the returned
  * port value.
  *
+ * This function delegates to `gpio_port_get_raw()` then converts line
+ * level to logic level.
```

Comment:
Anybody else have an opinion?  I can do this, though it will look weird if we define precise behavior for the `_raw()` functions and don't explain how the non-raw ones accomplish their task.